public/app/features/explore/Logs.tsx

```diff
@@ -56,7 +56,7 @@ function renderMetaItem(value: any, kind: LogsMetaKind) {
 }
 
 interface Props {
-  logRows?: LogRowModel[];
+  logRows: LogRowModel[];
```

Comment:
We used to return null in `<Logs />` if no `!logRows`. I have removed this condition to `LogsContainer` so it will return early for both - logs and live tailing and we don't have to do it in both components separately.